### ISSUE TITLE
RHS-C node has not enough free space in root partition

### ISSUE THREAD

**Issue by alram** [red-hat-storage/RHCS-vagrant#4]
mongod refuses to start with:
2017-04-18T21:04:30.272-0400 [initandlisten] ERROR: Insufficient free space for journal files
2017-04-18T21:04:30.272-0400 [initandlisten] Please make at least 3379MB available in /var/lib/mongodb/journal or use --smallfiles
2017-04-18T21:04:30.272-0400 [initandlisten]
2017-04-18T21:04:30.279-0400 [initandlisten] exception in initAndListen: 15926 Insufficient free space for journals, terminating

The space should be available since the root disk is 10GB but it looks like 4GB are not partitioned
[root@RHS-C mongodb]# lsblk
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
sda      8:0    0   10G  0 disk
└─sda1   8:1    0    6G  0 part /

**Comment by mulbc** [comment]
Good catch - I remember we had this in the past... Can you tell me exactly after which step this error appears? I think I would then just include an additional step in the ansible run: [rhsc-controller.yml](https://github.com/red-hat-storage/RHCS-vagrant/blob/RHSC/rhsc-controller.yml) that will configure mongodb with --smallfiles.
Changing the RHCS image (that is also the base for the Controller) would be more effort and would require everyone to re-fetch the image :/

**Comment by alram** [comment]
The playbook doesn't fail even though the skyring service doesn't start (due to mongod failing).
Starting with --smallfiles would mean modifying the unit file for the mongod service which might be tricky

**Comment by mulbc** [comment]
Proposal:
Ansible lineinfile that puts the '--smallfiles' into `/usr/lib/systemd/system/mongodb.service` (assuming that is the location?) and then we reload the service file with `systemctl daemon-reload` so that the changes are applied.
Two line change that saves everyone from re-downloading 1GB of image file and it is backwards compatible once I release a new image version which already has the fix
